src/transformers/models/xlm/modeling_tf_xlm.py

```diff
@@ -700,20 +699,20 @@ def __init__(self, config, *inputs, **kwargs):
     )
     def call(
         self,
-        input_ids=None,
-        attention_mask=None,
-        langs=None,
-        token_type_ids=None,
-        position_ids=None,
-        lengths=None,
-        cache=None,
-        head_mask=None,
-        inputs_embeds=None,
-        output_attentions=None,
-        output_hidden_states=None,
-        return_dict=None,
-        training=False,
-    ) -> Union[TFBaseModelOutput, Tuple[tf.Tensor]]:
+        input_ids: TFModelInputType | None = None,
+        attention_mask: np.ndarray | tf.Tensor | None = None,
+        langs: np.ndarray | tf.Tensor | None = None,
+        token_type_ids: np.ndarray | tf.Tensor | None = None,
+        position_ids: np.ndarray | tf.Tensor | None = None,
+        lengths: np.ndarray | tf.Tensor | None = None,
```

Comment:
I think it's okay to just say `tf.Tensor` here, even though `np.ndarray` inputs are accepted and implicitly converted too.

Reply:
Yeah, I don't have a strong opinion here as, to the top off my head, there's a nice mix of these through the TF models, some just show `tf.Tensor` and some others show the `ndarray` bit. 
Specifically, in this model, all the annotated `call`s are with the latter so for consistency we should leave it but then we will be falling short in the overall consistency that, as you say, `np.ndarray` are implicitly converted :thinking: